src/components/RenderRouter/VideoPlayerLive.tsx

```diff
@@ -29,7 +30,8 @@ export default class VideoPlayer extends React.Component<Props, State> {
       content: props.content,
       data: props.data,
       isLive: false,
-      liveEventJson: null
+      liveEventJson: null,
+      sunday: moment().tz("America/Toronto").isoWeekday(7).format('YYYY-MM-DD')
```

Comment:
I think something like `nextSundayDate` or `thisSundayDate` would be more clear.

Reply:
The ambiguity this vs next was what made me not do this in the first place. I'll go with `currentSundayDate`.